docs/api-testing/build-from-spec.md

```diff
@@ -2,25 +2,22 @@
 id: build-from-spec
 title: Building an API Test from a Spec File
 sidebar_label: Build Test from Spec File
-description: "The available spec files you can automatically generate from are: Swagger, RAML, OpenAPI 3, API Blueprint, I/O Docs, SOAP WSDL, and Postman Collection."
+description: "Automatically generate tests from a spec file in one of the following formats: Swagger, OpenAPI 3, and Postman Collection."
```

Comment:
@kimsauce Automatically generate tests from a spec file in one of the following formats: OpenAPI 3, HAR file and Postman Collection (btw, the only spec file is OpenAPI 3, the other files are not really spec files)

Reply:
Good point!